.ruby-version

```diff
@@ -1 +1 @@
-2.5.0
+2.5.1
```

Comment:
Let’s go to 2.6.1 whilst we’re making this change

Reply:
We should probably leverage travis' multi-ruby-version matrix capabilities and test against all Rails 5.2 supported versions of Ruby